### DISCUSSION TITLE
Can the localization of the language be made more convenient?

### DISCUSSION THREAD

**Discussion by zhuyifang** [directus/directus#7001] [Feature Requests]
For example, provide a language pack that can cover the built-in translation of the system. If you want more translation support, you can provide a way to upload the language pack, so that people who do good localization can facilitate external sharing, and the project can also Get better localization support

**Comment by rijkvanzanten** [comment]
Have you seen https://locales.directus.io? 😄 That's our translations tool that allows you to easily update the translations that are used in the app

**Reply by zhuyifang** [reply]
I have used it, but there are two doubts:
1. A lot of the content to be translated lacks enough context, so the translation results appear blunt and sloppy.
2. I don’t know how to quickly apply the translation content I have done to the project

**Reply by benhaynes** [reply]
CC: @jacobrienstra ❤️  (we should ensure all translation strings have context in Crowdin)

**Reply by zhuyifang** [reply]
benhaynes, thank you for your reply。
In addition, I think for example: the word "create" is actually enough in many scenes. Only when there are multiple types of content that cannot be created in one scene, it is necessary to be clear: create XXX

If adjusted in this way, the quality of translation work will be significantly improved.

**Reply by jrvidotti** [reply]
@benhaynes that's a nice contribution from @zhuyifang. We were discussing on Discord about custom title formatting according to language. This is nice to wrap it up with this idea. The language packs can be made individually downloadable and can comprehend the translation file and also conjunctions, prepositions, acronyms and a new dictionary to auto-correct accented words used by format-title package. 

If a translation adjustment is needed, admin can do it modifying the language pack and Directus codebase will not got bloated.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
